client/app/components/pages/Manuscripts.component.ts

```diff
@@ -1,5 +1,5 @@
 import { Component } from '@angular/core';
-import { ManuscriptService } from '../../services/menuscript.service';
+import { ManuscriptsService } from '../../services/menuscript.service';
```

Comment:
if you can change the file name to be manuscripts.service too that will be perfect!